routes/auths.js

```diff
@@ -14,4 +15,12 @@ router.post('/', validateLoginCredentials, async (req, res, next) => {
   }
 });
 
+router.post('/add', async (req, res) => {
+  const user = req.body;
+  const addeduser = await shoutout.create(user);
+  res.json(addeduser);
+  console.log(addeduser);
```

Comment:
You may have forgotten to remove the `console.log(addeduser);`

Reply:
None of these are my code so I'm not sure what to do with them.